### PR TITLE
Bug/ui tweaks

### PR DESCRIPTION
**Pull request by brianpeiris**
- "Next" button is too low on mic confirmation page
- "Enter VR" button is too high
- Increase info dialog padding to 30px
- De-emphasize profile header buttons
- Mouse drag sometimes drags selected ui items
- Move fullscreen call to just before enter scene
- Avoid flashing mic-denied icon
- Avoid continuous re-render on mic level

### CODE REVIEW

File: src/react-components/ui-root.js [hunk 1]
```diff
@@ -635,34 +635,41 @@ class UIRoot extends Component {
       ) : null;
 
     const micPanel =
-      this.state.entryStep === ENTRY_STEPS.mic_grant || this.state.entryStep == ENTRY_STEPS.mic_granted ? (
+      this.state.entryStep === ENTRY_STEPS.mic_grant || this.state.entryStep === ENTRY_STEPS.mic_granted ? (
         <div className="mic-grant-panel">
-          <div className="mic-grant-panel__title">
```
Comment (brianpeiris) [inline src/react-components/ui-root.js]: Mostly indentation differences below this line. I'd recommend the white space insensitive diff: https://github.com/mozilla/hubs/pull/290/files?w=1

File: src/assets/stylesheets/2d-hud.scss [hunk 2]
```diff
@@ -6,6 +6,7 @@
   align-items: center;
   height: 80px;
   width: 100%;
+  user-select: none;
```
Comment (gfodor) [inline src/assets/stylesheets/2d-hud.scss]: i think i've concluded that there is an infinite number of CSS attributes I do not know about

File: src/react-components/profile-entry-panel.js [hunk 3]
```diff
@@ -87,6 +87,7 @@ class ProfileEntryPanel extends Component {
                 onFocus={e => e.target.select()}
                 onChange={e => this.setState({ displayName: e.target.value })}
                 required
+                spellCheck="false"
```
Comment (gfodor) [inline src/react-components/profile-entry-panel.js]: looks like someone needs to install the ornithology pack to their system dictionary

File: src/react-components/ui-root.js [hunk 4]
```diff
@@ -675,100 +682,104 @@ class UIRoot extends Component {
...
-                srcSet="../assets/images/speaker_level@2x.png 2x"
-                className="audio-setup-panel__levels__icon-part"
-              />
+                {this.state.audioTrack ? (
+                  <img
+                    src="../assets/images/mic_level.png"
+                    srcSet="../assets/images/mic_level@2x.png 2x"
+                    className="audio-setup-panel__levels__icon-part"
+                  />
+                ) : (
```
Comment (gfodor) [inline src/react-components/ui-root.js]: It looks like something changed in this PR that causes me to see the "mic denied" image for a few frames before the audio track becomes non-null.

File: src/assets/stylesheets/profile.scss [hunk 5]
```diff
@@ -101,6 +102,8 @@
       &__icon {
 	@extend %fa-icon-button-icon;
 	background: transparent;
```
Comment (gfodor) [inline src/assets/stylesheets/profile.scss]: I was suggesting we drop the border altogether and leave the icon there fully-white -- the borders on these tiny icons are just very visually noisy. (I'm starting to think we should just drop them in the footer menu too, but I think they don't look too bad there.) The downside of removing the borders is just that the border has been the "affordance" (if you can call it that) that it's a button. But it kind of breaks down on these very small buttons.

Comment (gfodor) [reply src/assets/stylesheets/profile.scss]: Actually after playing around with it, I think just changing this to have `color: $light-grey` looks pretty good

File: src/assets/stylesheets/audio.scss [hunk 6]
```diff
@@ -1,10 +1,17 @@
...
   text-align: center;
   overflow-y: auto;
+  display: flex; 
+  flex-direction: column;
+  height: 100%;
   
+  &__enter-button-container {
+    flex: 1;
+    display: flex;
+    justify-content: center;
```
Comment (gfodor) [inline src/assets/stylesheets/audio.scss]: I think all the grant and enter containers should have a bottom margin of 24px, right now the buttons are too flush with the bottom of the dialog imho

File: src/assets/stylesheets/profile.scss (hunk 5 above)
Comment (gfodor) [reply src/assets/stylesheets/profile.scss]: basically if you change `color: $light-grey` these buttons have a new previously unused visual style -- but I think its fine. they are effectively "secondary action buttons" -- I could imagine us re-using that elsewhere. (For example, we may want to restyle the "hamburger menu" button this way)

**Pull request merged**